app/lib/actions/katello/repository_set/disable_repository.rb

```diff
@@ -21,17 +20,10 @@ def plan(product, content, options)
 
         private
 
-        def repository_mapper(product, content, substitutions, registry_name)
-          if content.content_type == ::Katello::Repository::CANDLEPIN_DOCKER_TYPE
-            ::Katello::Candlepin::DockerRepositoryMapper.new(product,
-                                                             content,
-                                                             registry_name)
-
```

Comment:
Note this completely removes the docker repository mapper since we don't use it.  It was intended to be used when ssl protected docker repos came online, but that hasn't happened and its been ~2 years.  I do not think there is any plan for this anymore.